src/gui/sdlmain.cpp

```diff
@@ -3143,6 +3143,10 @@ static void set_output(Section* sec, const bool wants_aspect_ratio_correction)
 	GFX_DisengageRendering();
 	// it's the job of everything after this to re-engage it.
 
+	auto set_texture_output_setting = []() {
+		set_section_property_value("sdl", "output", "texture");
+	};
```

Comment:
I know, but the whole thing is a mess. I just wanna do surgical improvements at this point, so this will stay (but enhancing it so it handles `opengl`->`texture` and `openglnb`->`texturenb` fallback correctly).